extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -347,7 +348,8 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   scrollHandler_() {
-    const currentScrollTop = this.element./*OK*/scrollTop;
+    // If scroll top is 0, it's set to 1 to avoid scroll-freeze issue.
+    const currentScrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
Hi @muxin, thanks for the review. I am gonna check all your comment and fix the code.